pilot/cmd/pilot-agent/app/cmd.go

```diff
@@ -109,7 +109,7 @@ func newProxyCommand() *cobra.Command {
 
 			raiseLimits()
 
-			proxy, err := initProxy(args)
+			err := initProxy(args)
```

Comment:
Would it make sense to keep using the new ProxyArgs type, but not make it a global? That seems like its going to make things harder to follow and test